schedule/yast/autoyast_create_hdd_gnome_RAID_encrypted_home.yaml

```diff
@@ -0,0 +1,22 @@
+name: autoyast_create_hdd_gnome_RAID_encrypted_home
+description:    >
+  Test performs autoyast installation to test encrypted home partition on RAID.
+conditional_schedule:
+  svirt_upload_assets:
+    ARCH:
+      s390x:
+        - shutdown/svirt_upload_assets
+vars:
+  AUTOYAST: yam/autoyast/create_hdd_gnome_x86_64_RAID_encrypted_home.xml
```

Comment:
we don't put vars in the schedules anymore, they should go to job group yaml, also advantage is that you can click and view those in openQA settings tab, but in general is for better organization.

Reply:
Removed from the schedule file and set it in test suite.